app/src/androidTest/assets/features/publicPrivateKeyGeneration.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Public and private key generation
+  As a user
+  I want to have the app generate a public/private key pair when I use it for the first time
+  So that I don't have to generate a pair myself
```

Comment:
> What the confirmation is trying to accomplish is a guarantee that I actually received your key and not someone else's key.
So we are exchanging them, I see that as a use of a key. Maybe that's where our definitions don't comply. Anyways this is a small detail to keep worrying over, keep this as you like @clenrock